test/plugins/test_lyrics.py

```diff
@@ -696,13 +696,27 @@ def test_fetch_synced_lyrics(self, mock_get):
         mock_get.return_value.json.return_value = mock_response
         mock_get.return_value.status_code = 200
 
+        self.plugin.config["synced"] = False
```

Comment:
I don't know if it's a problem with my setup, but I had to add these lines to the already existing tests to have them pass. It looks like the plugin config is mutated in one test case and then reused in the next, even though `setUp()` is called before each test which creates a new plugin instance and takes the config from there. But maybe I'm missing something.

If it is a setup (or logic) issue on my part, then having `config["synced"] = False` explicitly stated at least makes it clear what the expected configuration is for that part of the test. :)

Reply:
Can you create an issue about this? Just so it doesn't get forgotten and can be referenced.

Reply:
Done! #5133